src/components/LineChart/index.js

```diff
@@ -94,6 +96,8 @@ const defaultProps = {
   },
   crosshair: true,
   onMouseMove: null,
+  onMouseOut: null,
```

Comment:
Can you add a story for these? They just need to call `action('onMouseOut')` for example.